src/consts.js

```diff
@@ -9,6 +9,8 @@ module.exports = {
         appCmdQuit: "app.cmd.quit",
         appEventReady: "app.event.ready",
         appEventSecondInstance: "app.event.second.instance",
+        appExecuteJavaScript: "app.execute.javascript",
```

Comment:
Could you rename that to `appCmdExecuteJavascript: "app.cmd.execute.javascript"`?